[WIP]Change-Subject-field-to-a-multi-select-dropdown-#2499

neozenweb opened the pull request:
This PR adds a multiple select to the "Subject" field on the course creator form, replacing the text input.
![multi-select-subjects](https://user-images.githubusercontent.com/37545009/54784376-1784d800-4bfa-11e9-85bb-7408cc54c9fb.jpg)

neozenweb commented:
@bwreid , One of the Travis build seems to fail. How should I fix this problem? Thanks.

bwreid commented:
@neozenweb These errors are a bit odd. I'm going to re-run the build and we can see if anything changes. In the meantime, are you able to get the tests running successfully locally?

It still looks like this PR includes unnecessary file changes. The PR should only include the files necessary to solve this feature. There are a number of changes here that don't relate to the overall feature. You'll need to remove these from the PR first before we are able to merge.

neozenweb commented:
I rebased the code to master branch and just changed the one
file app/assets/javascripts/components/course_creator/course_form.jsx  so I
don't know where those changes are coming from. I also have run the jest
tests and the local testing successfully. Thanks.

On Fri, Mar 22, 2019 at 12:18 PM Wes Reid <notifications@github.com> wrote:

> @neozenweb <https://github.com/neozenweb> These errors are a bit odd. I'm
> going to re-run the build and we can see if anything changes. In the
> meantime, are you able to get the tests running successfully locally?
>
> It still looks like this PR includes unnecessary file changes. The PR
> should only include the files necessary to solve this feature. There are a
> number of changes here that don't relate to the overall feature. You'll
> need to remove these from the PR first before we are able to merge.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/2670#issuecomment-475683662>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AjzkMd2Kd4JarrthHcMxfj05wWV8ZP3Kks5vZQJAgaJpZM4cCXD0>
> .
>
